network/ocsp.go

```diff
@@ -0,0 +1,334 @@
+/*
+Copyright 2020, Cossack Labs Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package network
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/x509"
+	"errors"
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/crypto/ocsp"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+)
+
+// Errors returned by CRL verifier
+var (
+	ErrInvalidConfigOCSPRequired   = errors.New("Invalid `ocsp_required` value")
+	ErrInvalidConfigOCSPFromCert   = errors.New("Invalid `ocsp_from_cert` value")
+	ErrInvalidConfigAllRequiresURI = errors.New("When passing `--tls_ocsp_required=all`, URL is mandatory")
+	ErrOCSPRequiredAllButGotError  = errors.New("Cannot query OCSP server, but --tls_ocsp_required=all was passed")
+	ErrOCSPUnknownCertificate      = errors.New("OCSP server doesn't know about certificate")
+	ErrOCSPNoConfirms              = errors.New("None of OCSP servers confirmed the certificate")
+)
+
+// --tls_ocsp_required=<denyUnknown|allowUnknown|all>
+const (
+	// Deny certificates now known by OCSP server(s)
+	ocspRequiredYesStr = "denyUnknown"
```

Comment:
same, better to make them public and use in descriptions and tests instead hardcoded values

Reply:
> them public and use in descriptions and tests instead hardcoded values

yes, done just that, thanks